.travis.yml

```diff
@@ -9,13 +9,20 @@ cache:
 - directories:
   - $HOME/.cache
 
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - gcc-4.4
+      - g++-4.4
 install:
   - sudo apt-get install -y gdb
 
 script:
   - make
   - ulimit -c unlimited -S
-  - timeout --signal=SIGABRT 10m ./tst/webrtc_client_test
+  - if [ "$CC" != "gcc-4.4" ]; then timeout --signal=SIGABRT 10m ./tst/webrtc_client_test; fi
```

Comment:
@zhiyua-git You can remove this change, the `timeout` command isn't compiler specific. It just kills the test executable after 10 minutes.

It is useful for detecting deadlocks.

Reply:
We are using c++11 standard to write our unit/integ tests and that is not available in the old gcc compiler we use. Maybe I can try to only override CC instead of doing both. Will check if that is possible in another PR given this one is merged.